modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/SchedulerServiceActivator.java

```diff
@@ -60,6 +71,36 @@ public void activate( final BundleContext context )
         {
             LOG.debug( "RescheduleTask hasn't been started.", e );
         }
+
+        createConfigJobs( schedulerService );
+
+    }
+
+    private void createConfigJobs( final SchedulerService schedulerService )
+    {
+        adminContext().runWith( () -> schedulerConfig.jobs().forEach( job -> {
+            try
+            {
+                schedulerService.create( job );
+            }
+            catch ( Exception e )
+            {
+                LOG.debug( String.format( "Can't create [%s] job.", job.getName().getValue() ) );
```

Comment:
This is too broad to swallow.
Duplicate job exception should be swallowed.
Others should be well visible